.github/workflows/check-task-grunt.yml

```
@@ -0,0 +1,29 @@
+name: Check Task Grunt
+
+on: [push, pull_request, pull_request_target]
```

Entendi! Então é só deixar assim

```suggestion
on: [pull_request]
```

o problema de deixar assim é que só vai rodar quando abriu o PR, não quando alguém fizer um commit novo, sabe?

---

Eu acho que não tem problema deixar no push tbm

---

Saquei. Deixa como você alterou então!